python/doc/examples/meta_modeling/kriging_metamodel/plot_kriging_active_learning.py

```diff
@@ -0,0 +1,316 @@
+    boundsPoly.setLegend("95% bounds")
+
+    # Plot the function
+    graphFonction = ot.Curve(xGrid, yFunction)
+    graphFonction.setLineStyle("dashed")
+    graphFonction.setColor("magenta")
+    graphFonction.setLineWidth(2)
+    graphFonction.setLegend("Function")
+
+    thresholdFunction = ot.Curve([xMin, xMax], [event.getThreshold()] * 2)
+    thresholdFunction.setLineStyle("dashed")
+    thresholdFunction.setColor("red")
+    thresholdFunction.setLineWidth(2)
+    thresholdFunction.setLegend("Threshold")
+
+    # Draw the X and Y observed
+    cloudDOE = ot.Cloud(X, Y)
+    cloudDOE.setPointStyle("circle")
+    cloudDOE.setColor("red")
+    cloudDOE.setLegend("Data")
+    # Assemble the graphics
+
+    graph = ot.Graph()
+    graph.add(boundsPoly)
+    graph.add(graphFonction)
+    graph.add(thresholdFunction)
+    graph.add(cloudDOE)
+    graph.add(graphKriging)
+    graph.setLegendPosition("lower right")
+    graph.setAxes(True)
+    graph.setGrid(True)
+    graph.setTitle(
+        "Estimated probability = %f, Reference probability =  %f" % (proba, refProbability)
+    )
+    graph.setXTitle("X")
+    graph.setYTitle("Y")
+    return graph
+
+
+# %%
+# We start by creating the initial Kriging surrogate model :math:`\hat{\mathcal{M}}` on the 4 points in the design of experiments.
+# We estimate the probability on this surrogate model and compare with the reference probability computed on the real limit state function.
+
+# %%
+gprResult = createMyBasicKriging(X, Y)
+graph = plotMyBasicKriging(gprResult, xMin, xMax, X, Y, event, sampleX, probability)
+view = viewer.View(graph)
+
+
+# %%
+# Active learning Kriging to sequentially add new points
+# ------------------------------------------------------
+
+# %%
+# To sequentially add the new points, the "U criterion" is used.
+# It consists in finding the new point as the sample :math:`\mathbf{x}` in the Monte-Carlo experiment that minimizes  the following expression:
+# :math:`\frac{ \left| T - \hat{\mathcal{M}} ( \mathbf{x}) \right|}{\hat{\sigma}(\mathbf{x})}`
+# with :math:`\hat{\sigma}(\mathbf{x})` the square root of the marginal covariance of the Gaussian Process evaluated on :math:`\mathbf{x}`, and :math:`T` the event threshold (here 1.5)
+
+# %%
+def getNewPoint(X, gprResult, threshold):
+    """
+    Returns a new point to be added to the design of experiments.
+    This point maximizes the U criterion.
+    """
+    gpcc = otexp.GaussianProcessConditionalCovariance(gprResult)
+    response = gprResult.getMetaModel()(X)
+    conditionalVariance = gpcc.getConditionalMarginalVariance(X)
+
+    criterion = np.abs(
+        ot.Sample([ot.Point([event.getThreshold()])] * X.getSize()) - response
+    ) / np.sqrt(conditionalVariance + 1e-12)
+
+    iMaxU = int(np.argmin(criterion))
+    xNew = X[iMaxU]
+    return xNew
+
+
+# %%
+# We first call `getNewPoint` to get a point to add to the design of experiments.
+
+
+# %%
+xNew = getNewPoint(sampleX, gprResult, event.getThreshold())
+xNew
```

Comment:
```suggestion
print(xNew)
```
Improves the rendering IMO.